rest_framework/throttling.py

```diff
@@ -131,19 +131,26 @@ def allow_request(self, request, view):
             return self.throttle_failure()
         return self.throttle_success()
 
-    def throttle_success(self):
+    def add_request_to_history(self):
         """
         Inserts the current request's timestamp along with the key
         into the cache.
         """
         self.history.insert(0, self.now)
         self.cache.set(self.key, self.history, self.duration)
+
+    def throttle_success(self):
+        """
+        Called when a request to the API has passed throttling checks.
+        """
+        self.add_request_to_history()
```

Comment:
Probably rather just replicate the two lines in each of the two cases.

* Trivially obvious to ensure the two sets stay in sync.
* Removes one extra bit of indirection which aids comprehension.
* Smaller change footprint.
* Prevents us bloating out API that some folks will then end up relying on.